integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2516,8 +2513,8 @@ msgid ""
 "This is shown to the user when the mirrored part of a page is not available "
 "in their language."
 msgstr ""
-"Wird Benutzern angezeigt, wenn der gespiegelte Teil einer Seite nicht in "
-"ihrer Sprache verfügbar ist."
+"Wird Konten angezeigt, wenn der gespiegelte Teil einer Seite nicht in ihrer "
```

Comment:
```suggestion
"Wird Benutzer:innen angezeigt, wenn der gespiegelte Teil einer Seite nicht in ihrer "
```
Not sure why I missed this earlier, this is again on of the cases where users of the apps are addressed who do not have an account.

Reply:
No worries. I understand :)